pkg/etcd/etcd.go

```diff
@@ -512,7 +527,7 @@ func (e *ETCD) removePeer(ctx context.Context, id, address string) error {
 			}
 			if u.Hostname() == address {
 				if e.address == address {
-					logrus.Fatalf("node has been delete from the cluster. Backup and delete ${datadir}/server/db if you like to rejoin the node")
+					return fmt.Errorf("node has been delete from the cluster. Backup and delete ${datadir}/server/db if you like to rejoin the node")
```

Comment:
We're handling this automatically now, right? Under what circumstances would the user still need to manually delete these files?

Reply:
no not really, this change is only needed so that k3s doesnt exit when the node is deleted

Reply:
My only concern is that it's a file on a system that someone/something can modify outside the purview of etcd/k3s.